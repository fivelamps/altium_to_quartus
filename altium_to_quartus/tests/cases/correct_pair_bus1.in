NODENAME DATA.0_P$
    J2        72  U3       A1
NODENAME DATA.0_N$
    J2        72  U3       A2
NODENAME DATA.1_P$
    J2        70  U3       A3
NODENAME DATA.1_N$
    J2        70  U3       A4
NODENAME DATA.2_P$
    J2        72  U3       A5
NODENAME DATA.2_N$
    J2        72  U3       A6
NODENAME DATA.3_P$
    J2        70  U3       A7
NODENAME DATA.3_N$
    J2        70  U3       A8