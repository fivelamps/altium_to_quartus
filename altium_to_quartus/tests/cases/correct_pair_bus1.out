set_location_assignment PIN_A1 -to "DATA[0]"
set_location_assignment PIN_A2 -to "DATA[0](n)"
set_location_assignment PIN_A3 -to "DATA[1]"
set_location_assignment PIN_A4 -to "DATA[1](n)"
set_location_assignment PIN_A5 -to "DATA[2]"
set_location_assignment PIN_A6 -to "DATA[2](n)"
set_location_assignment PIN_A7 -to "DATA[3]"
set_location_assignment PIN_A8 -to "DATA[3](n)"
